# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lua/build.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lua-5.1/Makefile.orig	2006-02-16 16:45:17.000000000 +0100
+++ lua-5.1/Makefile	2006-03-08 16:48:29.000000000 +0100
@@ -48,9 +48,7 @@
 # Lua version. Currently used only for messages.
 V= 5.1
 
-all:	$(PLAT)
-
-$(PLATS) clean:
+all:
 	cd src && $(MAKE) $@
 
 test:	dummy
